groups/infrastructure/profiles/development-eu-west-2/cidev/vars

```diff
@@ -0,0 +1,5 @@
+environment = "cidev"
+aws_profile = "development-eu-west-2"
+enable_container_insights = true
```

Comment:
The variable is defined as `true` in all environment variables. It would be better to set the variable default to be `true` and the override var removed from all profiles.